doc/user_manual/code_interfaces/melcor.tex

```diff
@@ -109,17 +109,23 @@ \subsubsection{Models}
 \end{lstlisting}
 As it can be seen above, the \xmlNode{preexec} node must be specified, since MELCOR 2.1/2.2 must run the MELGEN utility
 code before executing. Once the \xmlNode{preexec} node is inputted, the execution of MELGEN is performed automatically by the Interface.
+Two addtional nodes need to be specified: the first one (i.e., \xmlNode{CodePlotFile}) indicates the name of the MELCOR plot file.
+The second one (i.e., \xmlNode{MelcorOutput}) specifies the name of the MELCOR message file.
```

Comment:
it seems that the interface accepts also a node "variables"

Reply:
added